### ISSUE TITLE
Support HTTP method Patch in netstandard environment

### ISSUE THREAD

**Issue by jzbrooks** [FacilityApi/FacilityCSharp#19]
Currently `[http(method: PATCH, path: "/foo")]` generates a mapping like:

```csharp
public static readonly HttpMethodMapping<FooRequestDto, FooResponseDto> FooMapping =
	new HttpMethodMapping<FooRequestDto, FooResponseDto>.Builder
	{
		HttpMethod = HttpMethod.Patch,
		Path = "/foo",
		ResponseMappings =
		{
			new HttpResponseMapping<FooResponseDto>.Builder
			{
				StatusCode = (HttpStatusCode) 200,
			}.Build(),
		},
	}.Build();
```

`HttpMethod.Patch` isn't available in netstandard as of netstandard 2.0. So this code breaks in that environment.

An alternative would be to use the constructor that accepts a string parameter instead of the static constants. i.e. `new HttpMethod("PATCH")` [here](https://github.com/FacilityApi/FacilityCSharp/blob/c74850b5f208d8883d96e6a538e6ab7664657a38/src/Facility.CodeGen.CSharp/CSharpGenerator.cs#L448).

**Comment by ejball** [comment]
Looks like I [changed it here](https://github.com/FacilityApi/FacilityCSharp/commit/8ccdabe9#diff-b659c4680eca73ae968ff6bae5b05d3f). I don't really remember why, unfortunately.

**Comment by ejball** [comment]
Fixed in 66705c1b86b3573ad7ec75027450bdb07a92ccda.
